sales.html

```diff
@@ -18,7 +20,7 @@ <h2>Capitol Hill Cookies</h2>
     <ul id="CapitolHill"></ul>
 
     <h2>Alki Cookies</h2>
-    <ul id="Alki"></ul>
+    <ul id="Alki"></ul> -->
```

Comment:
You shouldn't need any hard-coded markup inside your table, as you can append all necessary data from your app.js file.